### ISSUE TITLE
View full CloudFormation

### ISSUE THREAD

**Issue by Integralist** [apex/up#284]
Apologies if this is something that's already supported, but being able to view the full CloudFormation _before_ creating the stack would be nice. I

I'm aware that `up plan` will show changes to CloudFormation, but I want to see the full file right from the start. 

Thanks

**Comment by tj** [comment]
Some kind of option for that could be cool, I don't want to couple any of the commands too tightly with AWS/CloudFormation since it's just an implementation detail. Maybe displaying it when `--verbose`?

**Comment by olivoil** [comment]
Would you make it easy to include the cloudformation stack `up` generates into a higher-level CF template (or vice versa) ? Maybe output it to a json file?

Could be handy to manage the deployment of an app with only one CF stack, at least until #109 removes the need for a separate stack to provision other resources like dynamo tables, etc.

**Comment by tj** [comment]
@olivoil yeah I'd like to! #453 is one idea there, but I'd like to make it easy to use `up`'s stack tooling for custom stacks as well
